webui/react/.eslintrc.js

```diff
@@ -48,7 +48,11 @@ module.exports = {
     ],
     '@typescript-eslint/prefer-optional-chain': [ 'error' ],
     'array-bracket-spacing': [ 'error', 'always' ],
-    'array-element-newline': [ 'error', 'consistent' ],
+    'array-element-newline': [ 'error', {
+      ArrayExpression: 'consistent',
+      ArrayPattern: { multiline: true },
+    } ],
+    'arrow-parens': [ 'error', 'always' ],
```

Comment:
"as-needed" enforces no parens where they can be omitted.
?

Reply:
I was originally on the allowing them to be omitted, but as we write more code, I think forcing the parens is going to be beneficial with a larger team (allowing less variance for diffs in general)